packages/ui/src/components/ProjectCard/ProjectCard.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { Box, Flex, forwardRef, Grid } from '@chakra-ui/react';
+
+import { statusValueCheck } from '../../helpers/statusValueCheck';
+import { Button } from '../Button/Button';
```

Comment:
🙃 It should be enough to import it from '../Button' (similarly in the case of Typography)